examples/using-remark/src/pages/2016-04-15---hello-world-kitchen-sink/index.md

```diff
@@ -11,14 +11,14 @@ tags:
 ![](joshua-earle-234740.jpg)
 *Photo by [Joshua Earle](https://unsplash.com/@joshuaearle) via [Unsplash](https://unsplash.com/@joshuaearle?photo=-87JyMb9ZfU)*
 
-**_Note how all headlines below show an anchor link when you hover them?_**  
```

Comment:
Huh, I've been writing Markdown for years and never knew that.  *sigh*, will fix.